lab_2_keywords_cooccurrence/main.py

```diff
@@ -29,7 +47,19 @@ def extract_candidate_keyword_phrases(phrases: Sequence[str], stop_words: Sequen
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    key_phrases = []
+    for phrase in phrases:
+        lower_case = phrase.lower()
+        list_of_words_in_phrase = lower_case.split()
+        for word in list_of_words_in_phrase:
+            if word in stop_words:
+                index = list_of_words_in_phrase.index(word)
+                before_word = list_of_words_in_phrase[:index]
+                del list_of_words_in_phrase[index]
```

Comment:
why do you need this line?

overall, it is best not to modify the sequence you are iterating over
furthermore, it looks like you are going to save all the non-stop-words that came before that specific stop-words you are looking at, which is incorrect